README.rst

```diff
@@ -85,6 +85,9 @@ Setup (integrate in an existing django project)
         # admin
         'django.contrib.admin',
     ]
+    EXTENDED_APPS = ('django_netjsongraph',)
+
+Note that the ``EXTENDED_APPS`` may contain other names (if you installed other modules), in that case you should just add ``django_netjsongraph``.
```

Comment:
I think it should be `otherwise having just ``django_netjsongraph`` is enough`. What do you think @leonardomaccari?

Reply:
after you explained I got it